bedrock/firefox/views.py

```diff
@@ -215,9 +219,16 @@ def send_to_device_ajax(request):
     data_type = 'email' if '@' in phone_or_email else 'number'
     data[data_type] = phone_or_email
     form = SendToDeviceWidgetForm(data)
+
     if form.is_valid():
         phone_or_email = form.cleaned_data.get(data_type)
         platform = form.cleaned_data.get('platform')
+
+        # check for android and send to device test and update message if present
+        send_to_device_test = request.POST.get('android-send-to-device-test')
```

Comment:
This is unsaitized user-provided data. Should be validated to ensure it's one of the values we expect.